doc/bytecode.md

```diff
@@ -329,28 +336,29 @@ Verification
 ### from v1
 - `add64 reg, imm` can also use `r10` as destination register
 
-### until v2
+### except v2
 - Opcodes from the product / quotient / remainder instruction class are forbiden
 - `neg32` and `neg64` are allowed
 - `le` is allowed
 - `lddw` (opcodes `0x18` and `0x00`) is allowed
 - `hor64` is forbidden
-- `callx` source register is encoded in the imm field
+- `callx` target is encoded in the imm field
 
-### from v2
+### only in v2
 - Opcodes from the product / quotient / remainder instruction class are allowed
 - `neg32` and `neg64` are forbidden
 - `le` is forbidden
 - `lddw` (opcodes `0x18` and `0x00`) is forbidden
 - `hor64` is allowed
-- `callx` source register is encoded in the src field
+- `callx` target is encoded in the src field
 
 ### until v3
 - The targets of `call` instructions (which includes `syscall` instructions) are checked at runtime not verification time
 - The offset of jump instructions must be limited to the range of the bytecode
 
 ### from v3
 - Every function must end in a `ja` or `exit` instruction
+- `callx` target is encoded in the dst field
 - The targets of `call` instructions must have been registered at verification time
 - The targets of `syscall` instructions must have been registered at verification time
 - The offset of jump instructions must be limited to the range of the current function
```

Comment:
Are we going to have `hor64` with opcode `0x00`?

Regardless of whether we choose that or not, I believe everyone agreed that LDDW should cost two CUs. Shouldn't we take note of this cost difference here?